packages/@ourworldindata/grapher/src/timeline/TimelineComponent.tsx

```diff
@@ -988,6 +1177,27 @@ function SimpleTimeTooltip({
     return <div className="SimpleTimeTooltip">{formattedTime}</div>
 }
 
+/** Convert days-since-epoch integer to CalendarDate object */
+export function daysSinceEpochToCalendarDate(
+    daysSinceEpoch: number
+): CalendarDate {
+    const date = convertDaysSinceEpochToDate(daysSinceEpoch)
+    return new CalendarDate(
+        date.year(),
+        date.month() + 1, // CalendarDate is 1-indexed, dayjs is 0-indexed
+        date.date()
+    )
+}
+
+/** Convert CalendarDate to days-since-epoch integer */
+export function calendarDateToDaysSinceEpoch(date: CalendarDate): number {
+    const year = date.year
+    const month = date.month.toString().padStart(2, "0")
+    const day = date.day.toString().padStart(2, "0")
+    const isoString = `${year}-${month}-${day}`
```

Comment:
```suggestion
    const isoString = date.toString()
```

this'll always return a ISO string, see https://react-aria.adobe.com/internationalized/date/CalendarDate#to-a-string